src/NuGet.Core/NuGet.Packaging/ContentModel/ContentItemCollection.cs

```diff
@@ -277,6 +290,45 @@ private IList<ContentItem> FindItemsImplementation(PatternSet definition, IEnume
             return itemsList;
         }
 
+        private string GetRelatedFileExtensionProperty(string assemblyPath, IEnumerable<Asset> assets)
+        {
+            //E.g. if path is "lib/net472/A.B.C.dll", the prefix will be "lib/net472/A.B.C."
+            string assemblyPrefix = assemblyPath.Substring(0, assemblyPath.LastIndexOf('.') + 1);
+
+            if (_assemblyRelatedExtensions.TryGetValue(assemblyPrefix, out var relatedProperty))
+            {
+                return relatedProperty;
+            }
+
+            List<string> relatedFileExtensionList = null;
+            foreach (var asset in assets)
```

Comment:
Type in foreach loop:

```suggestion
            foreach (Asset asset in assets)
```

Reply:
Thanks! Fixed.